tools/api-tools/.editorconfig

```diff
@@ -0,0 +1,228 @@
+[*.{props,targets}]
```

Comment:
I think we can remove this file, and use the repo-level .editorconfig file instead.